### ISSUE TITLE
Error OnPlayerHurt

### ISSUE THREAD

**Issue by ArseeniK** [ServerMod/SCPDiscord#35]
```
OnPalyerHurt: Dahim (UNASSIGNED / 10) by Server (UNASSIGNED / 6) : Damage: 10 (DECONT) - Life before: 100
[19:34:35] [ERROR] [Event]  Event Handler: SCPDiscord.PlayerEventHandler Failed to handle event:Smod2.Events.PlayerHurtEvent
[19:34:35] [ERROR] [Event]  Array index is out of range.
[19:34:35] [ERROR] [Event]    at SCPDiscord.PlayerEventHandler.OnPlayerHurt (Smod2.Events.PlayerHurtEvent ev) [0x00000] in <filename unknown>:0
  at Smod2.Events.PlayerHurtEvent.ExecuteHandler (IEventHandler handler) [0x00000] in <filename unknown>:0
  at Smod2.Events.EventManager.HandleEvent[IEventHandlerPlayerHurt] (Smod2.Events.Event ev) [0x00000] in <filename unknown>:0
```

**Comment by KarlOfDuty** [comment]
This system is almost completely remade for the next update so we'll see if it still happens then.

**Comment by KarlOfDuty** [comment]
This issue was not fixed in the latest build.

**Comment by KarlOfDuty** [comment]
Issue was fixed in 0.2.2.
